src/pybind/mgr/dashboard/frontend/src/app/ceph/block/nvmeof-gateway-group/nvmeof-gateway-group.component.ts

```diff
@@ -0,0 +1,122 @@
+import { CdTableFetchDataContext } from '~/app/shared/models/cd-table-fetch-data-context';
+import { CdTableSelection } from '~/app/shared/models/cd-table-selection';
+import { Permission } from '~/app/shared/models/permissions';
+import { AuthStorageService } from '~/app/shared/services/auth-storage.service';
+import { Icons, IconSize } from '~/app/shared/enum/icons.enum';
+import { NvmeofGatewayGroup } from '~/app/shared/models/nvmeof';
+import { CephServiceSpec } from '~/app/shared/models/service.interface';
+
+@Component({
+  selector: 'cd-nvmeof-gateway-group',
+  templateUrl: './nvmeof-gateway-group.component.html',
+  styleUrls: ['./nvmeof-gateway-group.component.scss']
+})
+export class NvmeofGatewayGroupComponent implements OnInit {
+  @ViewChild(TableComponent, { static: true })
+  table: TableComponent;
+
+  @ViewChild('dateTpl', { static: true })
+  dateTpl: TemplateRef<any>;
+
+  @ViewChild('gatewayStatusTpl', { static: true })
+  gatewayStatusTpl: TemplateRef<any>;
+
+  permission: Permission;
+  tableActions: CdTableAction[];
+  columns: CdTableColumn[] = [];
+  selection: CdTableSelection = new CdTableSelection();
+  gatewayGroup$: Observable<CephServiceSpec[]>;
+  subject = new BehaviorSubject<CephServiceSpec[]>([]);
+  context: CdTableFetchDataContext;
+  gatewayGroupName: string;
+  subsystemCount: number;
+  gatewayCount: number;
+
+  icons = Icons;
+
+  iconSize = IconSize;
+
+  constructor(
+    public actionLabels: ActionLabelsI18n,
+    private authStorageService: AuthStorageService,
+    private nvmeofService: NvmeofService
+  ) {}
+
+  ngOnInit(): void {
+    this.permission = this.authStorageService.getPermissions().nvmeof;
+
+    this.columns = [
+      {
+        name: $localize`Name`,
+        prop: 'name'
+      },
+      {
+        name: $localize`Status`,
+        prop: 'statusCount',
+        cellTemplate: this.gatewayStatusTpl
+      },
+      {
+        name: $localize`Gateway nodes`,
+        prop: 'gateWayNode'
+      },
+      {
+        name: $localize`Subsystems`,
+        prop: 'subSystemCount'
+      },
+      {
+        name: $localize`Created on`,
+        prop: 'created',
+        cellTemplate: this.dateTpl
+      }
+    ];
+
+    this.gatewayGroup$ = this.subject.pipe(
+      switchMap(() =>
+        this.nvmeofService.listGatewayGroups().pipe(
+          switchMap((gatewayGroups: GatewayGroup[][]) => {
+            const groups = gatewayGroups?.[0] ?? [];
+            return forkJoin(
+              groups.map((group: NvmeofGatewayGroup) =>
+                this.nvmeofService.listSubsystems(group.spec.group).pipe(
+                  catchError(() => of([])),
+                  map((subs) => ({
+                    ...group,
+                    name: group.spec?.group,
+                    statusCount: {
+                      running: group.status?.running ?? 0,
+                      error: (group.status?.size ?? 0) - (group.status?.running ?? 0)
+                    },
```

Comment:
from the backend API itself we can do this fetch for subsystems and attach to response.
https://tracker.ceph.com/issues/74113